_security/configuration/security-admin.md

```diff
@@ -23,13 +23,13 @@ The `securityadmin.sh` script requires SSL/TLS HTTP to be enabled for your OpenS
 
 ## A word of caution
 
-If you make changes to the configuration files in `config/opensearch-security`, OpenSearch does _not_ automatically apply these changes. Instead, you must run `securityadmin.sh` to load the updated files into the index.
+If you make changes to the configuration files in `config/opensearch-security`, OpenSearch does _not_ automatically apply these changes. Instead, you must run `securityadmin.sh` to load the updated files into the index. The `securityadmin.sh` file can typically be found at `/tools/securityadmin.sh`
```

Comment:
For consistency-sake, can we make all paths relative to `<OPENSEARCH_HOME>` so this path would be `<OPENSEARCH_HOME>/plugins/opensearch-security/tools/securityadmin.[sh|bat]`